package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack",
+    "dev": "webpack serve"
+  },
```

Comment:
- [ ] It would be nice if you complete your scripts object by adding **start** key-value pair so the user can start your app using `npm run start` app easily 👍 Also it is a requirement for this milestone to help you in the future technologies 😊